core/src/main/java/apoc/cypher/CypherInitializer.java

```diff
@@ -75,6 +98,14 @@ public void available() {
         }).start();
     }
 
+    private String getMajorMinVersion(String completeVersion) {
+        if (completeVersion == null) {
+            return null;
+        }
+        final String[] split = completeVersion.split("\\.");
+        return split[0] + "." + split[1];
+    }
```

Comment:
Talked to @MishaDemianenko and there is no guarantee this is always on the major.minor format. Can we make it a bit more flexible and also add some sort of testing for this?

Reply:
I changed the handling. Now we check if `<KERNEL_VERSION> ` starts with `<APOC_VERSION>`,
because expect that APOC versioning is always consistent to Neo4j versioning,
e.g. if we have Neo4j `5_0_0`, the Apoc version will be `5_x_x_x` as well. 

And I added some tests too.